src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/ApplicationPropertyPage/ApplicationFrameworkValueProvider.cs

```diff
@@ -237,7 +252,7 @@ private async Task<bool> IsWPFApplicationAsync(IProjectProperties defaultPropert
                         await defaultProperties.SetPropertyValueAsync(StartupObjectMSBuild, "Sub Main");
                     }
 
-                    // Set the Application.xaml file's build action to None.
+                    // Set the Application.xaml file's build action to None and set Application Definition remove fpr Application.xaml.
```

Comment:
nit: typo

```suggestion
                    // Set the Application.xaml file's build action to None and set Application Definition remove for Application.xaml.
```

Also would this be clearer?

```suggestion
                    // Set the Application.xaml file's build action to None, which will also remove it from the ApplicationDefinition item group.
```

Probably the clearest comment would include examples of what happens in the project file XML.